scripts/in_container/prod/entrypoint_prod.sh

```diff
@@ -324,7 +324,7 @@ if [[ -n "${_PIP_ADDITIONAL_REQUIREMENTS=}" ]] ; then
     >&2 echo "         the container starts, so it is onlny useful for testing and trying out"
     >&2 echo "         of adding dependencies."
     >&2 echo
-    pip install --no-cache-dir --user "${_PIP_ADDITIONAL_REQUIREMENTS=}"
+    pip install --no-cache-dir --user ${_PIP_ADDITIONAL_REQUIREMENTS}
```

Comment:
It will work. We are using bash as interpreter and redirect only works when it is not passed via variable:

```
export _PIP_ADDITIONAL_REQUIREMENTS="pip<20.2.2 setuptools"
θ60° [jarek:~] 32s $ pip install --no-cache-dir --user ${_PIP_ADDITIONAL_REQUIREMENTS}
Collecting pip<20.2.2
  Downloading pip-20.2.1-py2.py3-none-any.whl (1.5 MB)
     |████████████████████████████████| 1.5 MB 1.2 MB/s 
Requirement already satisfied: setuptools in ./.pyenv/versions/3.7.9/lib/python3.7/site-packages (47.1.0)
Installing collected packages: pip
  WARNING: Value for scheme.headers does not match. Please report this to <https://github.com/pypa/pip/issues/9617>
  distutils: /home/jarek/.local/include/python3.7m/pip
  sysconfig: /home/jarek/.local/include/python3.7/pip
  WARNING: Additional context:
  user = True
  home = None
  root = None
  prefix = None
WARNING: Value for scheme.headers does not match. Please report this to <https://github.com/pypa/pip/issues/9617>
distutils: /home/jarek/.local/include/python3.7m/UNKNOWN
sysconfig: /home/jarek/.local/include/python3.7
WARNING: Additional context:
user = True
home = None
root = None
prefix = None
Successfully installed pip-20.2.1
WARNING: You are using pip version 21.1; however, version 21.1.2 is available.
You should consider upgrading via the '/home/jarek/.pyenv/versions/3.7.9/bin/python3.7 -m pip install --upgrade pip' command.
θ61° [jarek:~] 41s $ pip install --no-cache-dir --user ${_PIP_ADDITIONAL_REQUIREMENTS}

```